### PR TITLE
Blacklist ui

### LINKED ISSUE #1704: Simple UI for blacklist
- [ ] administration page for viewing blacklist

### PR DESCRIPTION
**Pull request by opqdonut**
closes #1704

# Definition of Done / Review checklist

## Reviewability
- [x] link to issue
- [x] consider adding screenshots for ease of review

## Documentation
- [x] update changelog if necessary

## Accessibility
- [x] all fields have a label

### CODE REVIEW

**Comment by opqdonut** [comment]
blacklist

**Comment by opqdonut** [comment]
blacklist2

File: src/cljs/rems/administration/administration.cljs [hunk 1]
```diff
@@ -63,7 +66,10 @@
                                                         :rems.administration/license
                                                         :rems.administration/create-license}
                                                       selected)
-                                       :active)]]])
+                                       :active)]]
+   [to-blacklist [:nav-item :nav-link (when (contains? #{:rems.administration/blacklist}
```
Comment (Macroz) [inline src/cljs/rems/administration/administration.cljs]: I think there's some possibility for refactoring to remove duplication and tedium.

Comment (opqdonut) [reply src/cljs/rems/administration/administration.cljs]: done

**Pull request merged**